### DISCUSSION TITLE
Figure out loading sp maps as mp without renaming source file

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#597] [Research]
Basically Titanfall2 seems to have some logic that handles maps differently depending on whether they start with `mp_` or `sp_`. We should figure out how to load `sp_` as `mp_` maps without having to re-ship all single player maps in Northstar as `mp_`.

**Comment by GeckoEidechse** [comment]
This is the stuff I remember from the top of my head, someone please add more info about this here.

**Comment by H0L0theBard** [comment]
I remember @wolf109909 attempting this at one point. 
Might be useful to ask him about it

**Comment by ASpoonPlaysGames** [comment]
Should be relatively simple to reverse engineer tbh, since we are just looking for where it checks `_mp`/`_sp`

**Comment by wolf109909** [comment]
I have tried to do that a year ago, but It isn't successful. I managed to make the game load mp scripts while loading sp_ maps, but there are more problems in resources so I didn't continue
